libraries/rush-lib/src/api/ExperimentsConfiguration.ts

```diff
@@ -61,6 +61,12 @@ export interface IExperimentsJson {
    */
   phasedCommands?: boolean;
 
+  /**
+   * If true, rush install or rush update implicitly specify --ignore-scripts during pnpm install,
+   * and run install lifecycle scripts by pnpm rebuild --pending after pnpm install successfully.
+   */
+  deferredInstallationScripts?: boolean;
```

Comment:
Should this be added as an official feature instead?

Reply:
Yes, I think it can be an official feature.  At the time when @chengcyber originally implemented it, it was an untested idea, but we have been using it successfully in our monorepo for over a year.

We can move this setting to `pnpm-config.json`

Reply:
Fixed